odoo/fields.py

```diff
@@ -1751,11 +1751,10 @@ def _get_stored_translations(self, record):
     def write(self, records, value):
         if not self.translate or value is False or value is None:
             return super().write(records, value)
-        cache = records.env.cache
-        cache_value = self.convert_to_cache(value, records)
-        records = cache.get_records_different_from(records, self, cache_value)
         if not records:
             return records
```

Comment:
If you remove the "reduction" of `records` above, then these two lines are useless.